### DISCUSSION TITLE
Hide Default Calendar?

### DISCUSSION THREAD

**Discussion by ykuzmycz** [MMRIZE/MMM-CalendarExt3#43] [General]
Fully appreciate that the default calendar module is needed.  Is there a way to hide it on the screen and just show the monthly calendar?

**Comment by eouia** [comment]
Remove `position` of `calendar` module in your configuration.

**Reply by ykuzmycz** [reply]
Thanks. I ended up hiding it via css. Did the trick via custom.css
